### PR TITLE
 update major subject areas

### PR DESCRIPTION
**Pull request by NuclearRedeye**
#### Background

Updates to the Major Subject Areas
 * Add 'Medicine' MSA
 * Add 'Human Biology' MSA
 * Remove 'Human Biology and Medicine' MSA'

#### Any relevant tickets

N/A

#### How has this been tested?

- [x] Unit / Integration tests
- [x] Browser tests
- [ ] End2end tests

#### UI Changes

- [ ] Has been reviewed against Designs
- [ ] Necessary updates to styleguide

### CODE REVIEW

File: config/__tests__/__snapshots__/config.ci.test.js.snap [hunk 1]
```diff
@@ -32,14 +32,15 @@ Array [
   "client.majorSubjectAreas.epidemiology-global-health",
   "client.majorSubjectAreas.evolutionary-biology",
   "client.majorSubjectAreas.genetics-genomics",
-  "client.majorSubjectAreas.human-biology-medicine",
```
Comment (nlisgo) [inline config/__tests__/__snapshots__/config.ci.test.js.snap:35]: Are we certain we want to remove `human-biology-medicine` and accept the impact, however small, on any manuscripts in various phases of submission?

File: config/__tests__/__snapshots__/config.ci.test.js.snap [hunk 2]
```diff
@@ -32,14 +32,15 @@ Array [
...
   "client.majorSubjectAreas.immunology-inflammation",
   "client.majorSubjectAreas.microbiology-infectious-disease",
   "client.majorSubjectAreas.neuroscience",
   "client.majorSubjectAreas.physics-living-systems",
   "client.majorSubjectAreas.plant-biology",
   "client.majorSubjectAreas.stem-cells-and-regenerative-medicine",
   "client.majorSubjectAreas.structural-biology-molecular-biophysics",
+  "client.majorSubjectAreas.human-biology",
+  "client.majorSubjectAreas.medicine",
```
Comment (nlisgo) [inline config/__tests__/__snapshots__/config.ci.test.js.snap]: Current order seems to be alphabetical. Should we retain that order?

File: config/__tests__/__snapshots__/config.default.test.js.snap [hunk 3]
```diff
@@ -32,14 +32,15 @@ Array [
...
-  "client.majorSubjectAreas.human-biology-medicine",
   "client.majorSubjectAreas.immunology-inflammation",
   "client.majorSubjectAreas.microbiology-infectious-disease",
   "client.majorSubjectAreas.neuroscience",
   "client.majorSubjectAreas.physics-living-systems",
   "client.majorSubjectAreas.plant-biology",
   "client.majorSubjectAreas.stem-cells-and-regenerative-medicine",
   "client.majorSubjectAreas.structural-biology-molecular-biophysics",
+  "client.majorSubjectAreas.human-biology",
+  "client.majorSubjectAreas.medicine",
```
Comment (nlisgo) [inline config/__tests__/__snapshots__/config.default.test.js.snap]: Current order seems to be alphabetical. Should we retain that order?

File: config/__tests__/__snapshots__/config.development.test.js.snap [hunk 4]
```diff
@@ -35,14 +35,15 @@ Array [
...
-  "client.majorSubjectAreas.human-biology-medicine",
   "client.majorSubjectAreas.immunology-inflammation",
   "client.majorSubjectAreas.microbiology-infectious-disease",
   "client.majorSubjectAreas.neuroscience",
   "client.majorSubjectAreas.physics-living-systems",
   "client.majorSubjectAreas.plant-biology",
   "client.majorSubjectAreas.stem-cells-and-regenerative-medicine",
   "client.majorSubjectAreas.structural-biology-molecular-biophysics",
+  "client.majorSubjectAreas.human-biology",
+  "client.majorSubjectAreas.medicine",
```
Comment (nlisgo) [inline config/__tests__/__snapshots__/config.development.test.js.snap]: Current order seems to be alphabetical. Should we retain that order?

File: config/__tests__/__snapshots__/config.end2end.test.js.snap [hunk 5]
```diff
@@ -32,14 +32,15 @@ Array [
...
-  "client.majorSubjectAreas.human-biology-medicine",
   "client.majorSubjectAreas.immunology-inflammation",
   "client.majorSubjectAreas.microbiology-infectious-disease",
   "client.majorSubjectAreas.neuroscience",
   "client.majorSubjectAreas.physics-living-systems",
   "client.majorSubjectAreas.plant-biology",
   "client.majorSubjectAreas.stem-cells-and-regenerative-medicine",
   "client.majorSubjectAreas.structural-biology-molecular-biophysics",
+  "client.majorSubjectAreas.human-biology",
+  "client.majorSubjectAreas.medicine",
```
Comment (nlisgo) [inline config/__tests__/__snapshots__/config.end2end.test.js.snap]: Current order seems to be alphabetical. Should we retain that order?

File: config/__tests__/__snapshots__/config.prod.test.js.snap [hunk 6]
```diff
@@ -32,14 +32,15 @@ Array [
...
-  "client.majorSubjectAreas.human-biology-medicine",
   "client.majorSubjectAreas.immunology-inflammation",
   "client.majorSubjectAreas.microbiology-infectious-disease",
   "client.majorSubjectAreas.neuroscience",
   "client.majorSubjectAreas.physics-living-systems",
   "client.majorSubjectAreas.plant-biology",
   "client.majorSubjectAreas.stem-cells-and-regenerative-medicine",
   "client.majorSubjectAreas.structural-biology-molecular-biophysics",
+  "client.majorSubjectAreas.human-biology",
+  "client.majorSubjectAreas.medicine",
```
Comment (nlisgo) [inline config/__tests__/__snapshots__/config.prod.test.js.snap]: Current order seems to be alphabetical. Should we retain that order?

File: config/__tests__/__snapshots__/config.staging.test.js.snap [hunk 7]
```diff
@@ -32,14 +32,15 @@ Array [
...
-  "client.majorSubjectAreas.human-biology-medicine",
   "client.majorSubjectAreas.immunology-inflammation",
   "client.majorSubjectAreas.microbiology-infectious-disease",
   "client.majorSubjectAreas.neuroscience",
   "client.majorSubjectAreas.physics-living-systems",
   "client.majorSubjectAreas.plant-biology",
   "client.majorSubjectAreas.stem-cells-and-regenerative-medicine",
   "client.majorSubjectAreas.structural-biology-molecular-biophysics",
+  "client.majorSubjectAreas.human-biology",
+  "client.majorSubjectAreas.medicine",
```
Comment (nlisgo) [inline config/__tests__/__snapshots__/config.staging.test.js.snap]: Current order seems to be alphabetical. Should we retain that order?

File: config/__tests__/__snapshots__/config.test.test.js.snap [hunk 8]
```diff
@@ -34,14 +34,15 @@ Array [
...
-  "client.majorSubjectAreas.human-biology-medicine",
   "client.majorSubjectAreas.immunology-inflammation",
   "client.majorSubjectAreas.microbiology-infectious-disease",
   "client.majorSubjectAreas.neuroscience",
   "client.majorSubjectAreas.physics-living-systems",
   "client.majorSubjectAreas.plant-biology",
   "client.majorSubjectAreas.stem-cells-and-regenerative-medicine",
   "client.majorSubjectAreas.structural-biology-molecular-biophysics",
+  "client.majorSubjectAreas.human-biology",
+  "client.majorSubjectAreas.medicine",
```
Comment (nlisgo) [inline config/__tests__/__snapshots__/config.test.test.js.snap]: Current order seems to be alphabetical. Should we retain that order?

File: config/default.js [hunk 9]
```diff
@@ -82,6 +81,8 @@ module.exports = {
         'Stem Cells and Regenerative Medicine',
       'structural-biology-molecular-biophysics':
         'Structural Biology and Molecular Biophysics',
+      'human-biology': 'Human Biology',
+      medicine: 'Medicine',
```
Comment (nlisgo) [inline config/default.js]: Current order seems to be alphabetical. Should we retain that order?

Comment (NuclearRedeye) [reply config/default.js]: Good spot! Yes we should :-)

**Comment by nlisgo** [comment]
This looks good to me as long as we confirm that removal of `human-biology-medicine` is expected and non-disruptive for manuscript in progress.

**Review by will-byrne — APPROVED** [review]

**Comment by NuclearRedeye** [comment]
Maria has confirmed, it's just a rename. And to the best of my testing it seems that it should be non-disruptive.

**Review by nlisgo — APPROVED** [review]

**Review by giorgiosironi — APPROVED** [review]
Not completely familiar but as a MSA change it matches the specification

**Pull request merged**